ifupdown2/lib/iproute2.py

```diff
@@ -333,6 +333,9 @@ def link_create_vxlan(self, name, vxlanid, localtunnelip=None, svcnodeip=None,
         if learning == 'off':
             cmd.append("nolearning")
 
+        if udp_csum == 'off':
+            cmd.append("noudpcsum")
+
```

Comment:
Are you missing `tos` in this function?

Reply:
never mind the function is not used anymore so i guess thats fine :)

Reply:
I've added `tos` anyway, so it's there if this ends up being used in the future.